cmd/libs/go2idl/deepcopy-gen/generators/deepcopy.go

```diff
@@ -65,8 +65,7 @@ func Packages(context *generator.Context, arguments *args.GeneratorArgs) generat
 	inputs := sets.NewString(arguments.InputDirs...)
 	packages := generator.Packages{}
 	header := append([]byte(
-		`
-// +build !ignore_autogenerated
+		`// +build !ignore_autogenerated
```

Comment:
The newline at the beginning doesn't actually make a difference in output.  Same with the conversion one.